packages/odyssey-react/src/components/SearchInput/SearchInput.module.scss

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.affix {
```

Comment:
AFAIK these styles should also be in TextInput since we want them accessible by anything using an `affix`, not just Search.

Reply:
yes, that would be great. I would just need css that would work for both icon and button cases. I couldn't get right 😞